pkg/kube/client.go

```diff
@@ -32,7 +32,10 @@ import (
 	gatewayapiinformer "github.com/maistra/xns-informer/pkg/generated/gatewayapi"
 	istioinformer "github.com/maistra/xns-informer/pkg/generated/istio"
 	kubeinformer "github.com/maistra/xns-informer/pkg/generated/kube"
+	osrouteinformer "github.com/maistra/xns-informer/pkg/generated/openshift/route"
 	xnsinformers "github.com/maistra/xns-informer/pkg/informers"
+	osrouteclient "github.com/openshift/client-go/route/clientset/versioned"
+	osroutefake "github.com/openshift/client-go/route/clientset/versioned/fake"
```

Comment:
Can you rename these to just `route`, `routeinformer`? Also, why do we need the fake?